index.html

```diff
@@ -17,6 +18,13 @@
     'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
     })(window,document,'script','dataLayer','GTM-5BNWTHP');
     <!-- End Google Tag Manager -->
+       jQuery(document).ready(function(){
+        jQuery('a[href^="https://"]').each(function(){ 
+            var oldUrl = jQuery(this).attr("href"); 
+            var newUrl = oldUrl.replace("CONTENT-URL", window.location.host.split(".")[0]); 
+            jQuery(this).attr("href", newUrl); 
+        });
+    });
```

Comment:
This should be moved down to the line after <script src="javascripts/all.js"></script> and it should be put into a separate script tag.